jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/ProcessInstanceLogPresenter.java

```diff
@@ -88,34 +90,31 @@ public void init() {
     }
 
     public IsWidget getWidget() {
-        return view;
-    }
-
-    public String getProcessName() {
-        return processName;
-    }
-
-    public void setProcessName(String processName) {
-        this.processName = processName;
+        return ElementWrapperWidget.getWidget(view.getElement());
     }
 
     public void setServerTemplateId(String serverTemplateId) {
         this.serverTemplateId = serverTemplateId;
     }
 
-    public void refreshProcessInstanceData(final LogOrder logOrder,
-                                           final LogType logType) {
+    public void loadProcessInstanceLogs() {
         try {
             final FilterSettings currentTableSettings = dataSetQueryHelper.getCurrentTableSettings();
             currentTableSettings.setServerTemplateId(this.serverTemplateId);
-            currentTableSettings.setTablePageSize(pageSize);
-            dataSetQueryHelper.setLastOrderedColumn(COLUMN_LOG_DATE);
-            dataSetQueryHelper.setLastSortOrder(logOrder.equals(LogOrder.ASC) ? SortOrder.ASCENDING : SortOrder.DESCENDING);
-
+            currentTableSettings.setTablePageSize(PAGE_SIZE);
+
+            currentTableSettings.addSorOperation(new ColumnSort(COLUMN_LOG_DATE,
```

Comment:
we don't expect the sort order to change, so I'm thinking this should be done at the filtersettings creation time, it would avoid having to do these manipulations to the table settings, wdyt? _FilterSettingsBuilderHelper_ would need to be improved to accept multiple columns.